tests/functional/ldap/test_acquire_ldap.py

```diff
@@ -84,12 +85,33 @@ def mock_query_ldap(
     monkeypatch.setattr(sarc.ldap.read_mila_ldap, "query_ldap", mock_query_ldap)
 
     def mock_query_mymila(tmp_json_path):
-        return fake_mymila_data(nbr_users)
+        return fake_mymila_data(nbr_users, nbr_profs)
 
     monkeypatch.setattr(sarc.ldap.mymila, "query_mymila", mock_query_mymila)
 
     # Patch the built-in `open()` function for each file path
     with patch("builtins.open", side_effect=mock_file):
         sarc.ldap.acquire.run()
 
+    users = get_users()
+    assert len(users) == nbr_users
+
+    for i in range(nbr_profs):
+        user = users[i]
+        # A prof should not have a supervisor or co-supervisor but there's a
+        # mismatch between the number of profs in ldap (1) and the generated
+        # mymila data (5)
+        # assert user.mila_ldap["supervisor"] is None
+        # assert user.mila_ldap["co_supervisor"] is None
```

Comment:
Would it be worth it to harmonize `fake_raw_ldap_data` with `fake_mymila_data` to set the number of professor to sometimes be >1?